logger.go

```diff
@@ -0,0 +1,40 @@
+package minimemcached
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+type LogLevel string
+
+const (
+	// Info is the default LogLevel of Logger.
+	Info LogLevel = "info"
+	// Debug helps you debug mini-memcached usages,
+	//and logs all command and its result after being processed.
+	Debug LogLevel = "debug"
+	// Off does not log anything.
+	Off LogLevel = "off"
+)
+
+// Logger logs certain messages depending on the LogLevel.
+// TODO:(@sang-w0o): Selectively accept where log output will go to.
+type Logger struct {
```

Comment:
I believe Logger should be implemented as an interface.

Implementation in GORM can be a good reference. https://github.com/go-gorm/gorm/blob/master/logger/logger.go

I would suggest interface below.

```go
// Logger logger interface
type Logger interface {
	Info(string)
	Debug(string)
}
```

Furthermore,

1. Checking log level each time is inefficient
2. log levels are cascading. (e.g. INFO level logs should be printed in DEBUG log level)